src/native/external/brotli.cmake

```diff
@@ -30,3 +30,4 @@ target_compile_definitions(brotlicommon PRIVATE BROTLI_SHARED_COMPILATION BROTLI
 # Don't build the brotli command line tool unless explicitly requested
 # (which we never do)
 set_target_properties(brotli PROPERTIES EXCLUDE_FROM_ALL ON)
+set_property(TARGET brotlienc brotlidec brotlicommon PROPERTY INTERPROCEDURAL_OPTIMIZATION OFF)
```

Comment:
Is this going to disable LTCG for regular Brotli build that ships in System.Compression.Native as well?

Reply:
I've adjusted this to only apply in the coreclr reference (so singlefilehost and nativeaot).

Reply:
Is it too hard to keep LTCG for singlefilehost?

Reply:
Yes, our current build setup isn't conducive to building two flavors of our vendored libraries (we would need to be able to define separate target names and include the subdirectories twice).

There are alternative ways we could include the vendored libs, but that would require significant changes to how we build. I think that's something we can consider in .NET 10 but in a later PR (especially if we want this in P1).

Reply:
Could you please open an issue on this?